spec/connectors_sdk/sharepoint/extractor_spec.rb

```diff
@@ -58,7 +58,7 @@
     end
```

Comment:
Maybe worth adding one new test/expectation about what the `path` of these should be now? Just to protect against any regression.

Reply:
It was tested in adapter_spec. But I will add one test in this spec.